baker/archival/archiveChangedPages.ts

```diff
@@ -177,22 +180,32 @@ const getPostsToArchive = async (
         const postsToArchive = allChecksums.filter((checksum) =>
             postSlugs.includes(checksum.postSlug)
         )
-        const postIds = new Set(postsToArchive.map((post) => post.postId))
-        const imagesByPostIdToArchive = _.pickBy(imagesByPostId, (_, postId) =>
-            postIds.has(postId)
-        )
 
         if (postSlugs.length !== postsToArchive.length) {
             throw new Error(
                 `Not all post slugs were found in the database. Found ${postsToArchive.length} out of ${postSlugs.length}.`
             )
         }
 
-        return { postsToArchive, imagesByPostId: imagesByPostIdToArchive }
+        const postIds = new Set(postsToArchive.map((post) => post.postId))
+        const imagesByPostIdToArchive = _.pickBy(imagesByPostId, (_, postId) =>
+            postIds.has(postId)
+        )
```

Comment:
Just noticed this now, we can actually turn this into
```suggestion
        const imagesByPostIdToArchive = _.pick(imagesByPostId, [...postIds])
```